.lycheeignore

```diff
@@ -0,0 +1,93 @@
+# .lycheeignore
+
+# Global ignores for Lychee link checker
+# Docs: https://github.com/lycheeverse/lychee#ignoring-links
+#
+# Philosophy:
+# - Only ignore URLs that are inherently uncheckable (require auth, localhost, API-only)
+#   or consistently noisy (bot blocking, weird 4xx).
+# - Do NOT blanket-ignore normal public docs sites so we can still catch real regressions.
+
+#
+# Mintlify internal routes and static assets (root-relative paths)
+# These only become real URLs after Mintlify builds the site.
+# Examples:
+#   /intro/user-authentication
+#   /how-tos/check-google-calendar-availability
+#   /img/intro/universal_login.png
+#
+^/
```

Comment:
eep, does lychee actually ping on these if we don't ignore them explicitly? maybe a markdown processing effect? i do wonder if they have a native way to handle this.

Reply:
i'm on board with that. let's leave a persistent note to ourselves somewhere (github issues? ... jira??) for internal link checking with lychee in the future. it would be nice to toss some of this reasoning into the comment above this exclude in `.lycheeignore`, but otherwise feel free to resolve this thread.